autonomy/cli/deploy.py

```diff
@@ -139,6 +143,14 @@ def deploy_group(
     default=False,
     help="Use local tendermint chain setup.",
 )
+@click.option(
+    "-eap",
+    "--expose-agent-port",
+    "agent_ports",
+    multiple=True,
+    type=AgentPortParameter(),
+    help="Expose agent port.",
+)
```

Comment:
Please suggest better flag name if any, I don't think `-eap` is an optimal choice

Reply:
Running `autonomy deploy build ... -eap 0:8080:8081` will produce the following docker compose config

```yaml
  abci0:
    (...)
    ports:
      - 8080:80
```

Running `autonomy deploy build ... -eap 0:8080:8081 --kubernetes` will produce the following docker compose config

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: agent-node-1
spec:
  (...)
  template:
    metadata:
        (...)
    spec:
      containers:
        (...)
        env:
        (...)  
        image: valory/oar-hello_world:bafybeihuylwbdvru6evry6dv7r7mfuaasjclo3f5vnyes6cobkedybsqwy
        imagePullPolicy: Always
        name: aea
        ports:
        - containerPort: 8081
```


The expose agent port flag takes the input in `<AGENT_ID:HOST_PORT:CONTAINER_PORT>` format

Reply:
@8ball030 @0xArdi Please confirm these configs are correct

Reply:
I'm not sure doing it with a flag is the right way to go. 
We already have like 3-5 random flags you have to pass and adding more just makes it more complicated and requires more time

Reply:
I'm also more inclined to do it via config vars, since all the configuration is already passed like that. I'd say it's more consistent from the user perspective.

Reply:
so basically for now we would need that flag 4 times repeated (assuming 4 agents)